### DISCUSSION TITLE
Zone Entries

### DISCUSSION THREAD

**Discussion by tjwoodwa** [DeniseCaiLab/ezTrack#48] [Q&A]
Hello! Thanks for the great tool.  I am currently using the LocationTracking module to look at Elevated Plus Maze videos (similar to the example provided).  I am wondering if there is a way to easily include the # of entries into each zone in the summary data as well.  Right now I can see the proportion of time during each user-defined bin (1 minute in my case) as well as the total number of time the animal enters the zone (open arm or close arm in my case). 

I know one could probably write a script that looks at each animals 'LocationOutput' file and counts the number of TRUE sessions for each area. Is there something already built in to ezTrack like this that I don't know about? Thanks!

**Comment by ZachPenn** [comment]
This isn't currently built in but I think it would be a great thing to be added, and pretty simple to do.  I did recently add a column to the LocationOutput file/dataframe entitled `ROI_transition`, which indicates on which frames transitions occur, so if you index the data frame based upon this it should be pretty easy to get the count info you need until I formally add it (e.g. `transition_frames = location[location['ROI_transition']]`)
